style.css

```diff
@@ -0,0 +1,200 @@
+/* font-family: 'Work Sans', sans-serif;*/
+*{
+    box-sizing: border-box;
+    margin: 0%;
+}
+:root{
+    --clr-primary: #f70058;
+    --clr-dark: #0c0e36;
+    --clr-light: #fff;
+    --fw-bold: 400;
+    --fw-bold: 700;
+    --fw-black: 900;
+    --grid: 1fr;
+    --grid: minmax(1em,10%)
+            minmax(10rem,22ch)
+            minmax(10rem,23ch)
+            minmax(1em,3fr)
+            max-content;
+}
+body{
+    font-family: 'Work Sans' , sans-serif;
+    font-size: 1.125rem;
+
+    color: var(--clr-dark);
+    line-height: 1.6;
+}
+header{
+    background-color: var(--clr-light);
+    text-align: center;
+}
+
+.nav_list{
+    display: flex;
+    justify-content: space-between;
+
+    padding: 1em;
+    /*margin-top: 1em;*/
+
+    list-style-type: none;
+    
+    font-size: 14px;
+    text-transform: uppercase;
+    font-weight: var(--fw-bold);
+    letter-spacing: 1px;
+}
+.nav_list-item{
+    margin-left: 1em;
+}
+.nav_link{
+    text-decoration: none;
+    color: inherit;
+}
+.weather{
+    display: grid;
+    grid-template-columns: min-content 1fr;
+    
+    background: var(--clr-dark);
+    color: var(--clr-light);
+}
+.weather_icon{
+    grid-row: span 2;
+    background: var(--clr-primary);
+
+    padding: .5em;
+
+    display: flex;
+    align-items: center;
+}
+.weather_info{
+    margin-left: .75em;
+
+    text-align: left;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    font-size: 12px;
+
+    color: rgb(255,255,255,0.7);
+
+    display: flex;
+}
+.weather_info span{
+    color: var(--clr-light);
+    font-size: 15px;
+
+    margin-left: auto;
+    margin-right: .50rem;
+
+    text-transform: none;
+}
+.weather_info:first-of-type{
+    margin-top: .5em;
+    margin-bottom: .5rem;
+    position: relative;
+}
+.weather_info:last-of-type{
+    margin-bottom: .4em;
+
+}
+.weather_info:first-of-type::after{
+    content: '';
+    position: absolute;
+    
+    opacity: .25;
+    
+    height: 2px;
+    width: calc(100% - .75rem);
+    bottom: -.25rem;
+
+    background: var(--clr-light);
+}
+.main{
+    padding: 3em 1em 5em;
+}
+.main--ski{
+    background-image: url(../img/sebastian-staines-xZIFEPtRsRI-unsplash\ 1.png);
```

Comment:
your image is not visible you wrote wrong way just put this and background image will be visible

```js
    background-image: url('img/sebastian-staines-xZIFEPtRsRI-unsplash 1.png'); ```

Reply:
thanks..